composer.json

```diff
@@ -10,5 +10,10 @@
     "require": {
         "php": ">=5.5",
         "zendframework/zendframework": "~2.5"
+    },
+    "autoload": {
+        "psr-4": {
+          "Application\\": "module/Application/src/Application"
```

Comment:
No need for the second Application folder in the path. Jut use "Application\\": "module/Application/src" for psr-4